### PR TITLE
Fix mapped severity

### PR DESCRIPTION
**Pull request by Abuelodelanada**
This PR fixes: https://github.com/canonical/cos-proxy-operator/issues/146

We multiply the value  by 2 so the severity is mapped to a critical

## Testing Instructions

1. Create a k8s model and deploy `cos-lite` in it
2. Offer the `prometheus-scrape` interface: `juju offer prometheus:metrics-endpoint prometheus-scrape`
3. Create a machine model
4. Consume the `prometheus-scrape` offer: `juju consume microk8s:admin/cos.prometheus-scrape`
6.  Deploy this bundle:

```yaml
default-base: ubuntu@22.04/stable
saas:
  grafana-dashboards:
    url: microk8s:admin/cos.grafana-dashboards
  loki-logging:
    url: microk8s:admin/cos.loki-logging
  prometheus-receive-remote-write:
    url: microk8s:admin/cos.prometheus-receive-remote-write
  prometheus-scrape:
    url: microk8s:admin/cos.prometheus-scrape
applications:
  cos-proxy:
    charm: local:cos-proxy-0
    num_units: 1
    to:
    - "2"
    constraints: arch=amd64
  nrpe:
    charm: nrpe
    channel: latest/edge
    revision: 139
  ubu22:
    charm: ubuntu
    channel: latest/stable
    revision: 25
    num_units: 1
    to:
    - "3"
    constraints: arch=amd64
    storage:
      block: loop,100M
      files: rootfs,100M
machines:
  "2":
    constraints: arch=amd64
  "3":
    constraints: arch=amd64
relations:
- - cos-proxy:monitors
  - nrpe:monitors
- - cos-proxy:downstream-logging
  - loki-logging:logging
- - ubu22:juju-info
  - nrpe:general-info
- - cos-proxy:downstream-prometheus-scrape
  - prometheus-scrape:metrics-endpoint
```

7. Stop the nrpe-exporter service: `juju ssh cos-proxy/1 sudo systemctl stop nrpe-exporter`
8. Wait 15 minutes and check in Alertmanager:

image

### CODE REVIEW

File: lib/charms/grafana_agent/v0/cos_agent.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 Using the `COSAgentProvider` object only requires instantiating it,
 typically in the `__init__` method of your charm (the one which sends telemetry).
 
-The constructor of `COSAgentProvider` has only one required and nine optional parameters:
+The constructor of `COSAgentProvider` has only one required and ten optional parameters:
```
Comment (sed-i) [inline lib/charms/grafana_agent/v0/cos_agent.py:25]: We won't need to keep this comment up do date if for examplec we remove this sentence entirely.

```suggestion
```

Comment (Abuelodelanada) [reply lib/charms/grafana_agent/v0/cos_agent.py:25]: DONE: https://github.com/canonical/grafana-agent-operator/commit/00a71be918731428657f4e1f482386b17b5e0549

PR: https://github.com/canonical/grafana-agent-operator/pull/218

File: tests/scenario/test_alerts.py [hunk 2]
```diff
@@ -28,10 +30,16 @@ def test_relation(ctx, n_remote_units):
             for i in range(n_remote_units)
         },
     )
-    state_in = State(leader=True, relations=[monitors], networks={"monitors": Network.default()})
+    network = Network("monitors", [BindAddress([Address("192.0.2.1")])])
```
Comment (sed-i) [inline tests/scenario/test_alerts.py:33]: Iirc, with modern ops, we no longer need to mock the network ourselves?

File: tox.ini [hunk 3]
```diff
@@ -87,7 +87,7 @@ description = Run integration tests
 description = Run scenario tests
 deps =
     pytest
-    ops-scenario>=6
+    ops-scenario
```
Comment (sed-i) [inline tox.ini:90]: While we're at it, if not too much extra effort, can we change this to
```suggestion
    ops[testing]
```
?

**Review by sed-i — APPROVED** [review]

**Pull request merged**